### ISSUE TITLE
Terms of Use screen - Screen flow 

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#571]
At present (close button > switch to French > Terms of use> content).

It should be (close button > Terms of use > switch to French > content).
